deploy/kubernetes/dolphinscheduler/templates/statefulset-dolphinscheduler-worker.yaml

```
@@ -142,9 +142,11 @@ spec:
         - name: {{ include "dolphinscheduler.fullname" . }}-worker-logs
           emptyDir: {}
         {{- end }}
+        {{- if .Values.worker.enableCustomizedConfig }}
         - name: worker-config-volume
           configMap:
             name: {{ include "dolphinscheduler.fullname" . }}-worker
+        {{- end -}}
```

There is a difference between `{{- end }}` and `{{- end -}}`, and in this case, you should use `{{- end }}`.
```suggestion
        {{- end }}
```

---

sure, i will modify the others, too